internal/handler/msg_listcollections.go

```diff
@@ -78,26 +78,47 @@ func (h *Handler) MsgListCollections(ctx context.Context, msg *wire.OpMsg) (*wir
 	collections := types.MakeArray(len(res.Collections))
 
 	for _, collection := range res.Collections {
-		d := must.NotFail(types.NewDocument(
-			"name", collection.Name,
-			"type", "collection",
-		))
+		options := []any{}
+		info := []any{"readOnly", false}
```

Comment:
Well, in cases like that, you could start with an empty `types.Document` and use `Set` to add fields. But that does not matter much.